### ISSUE TITLE
Own Detection Modell

### ISSUE THREAD

**Issue by Schubert-Tom** [JaidedAI/EasyOCR#729]
Hi,

I am trying to train my own text detection modell on pictures with the following format:

![4](https://user-images.githubusercontent.com/64316635/169139850-6b662458-92ff-4d5c-acd0-abb801866d74.jpg)

![11](https://user-images.githubusercontent.com/64316635/169139499-0be687d5-c01f-4920-8d88-b824dad47203.jpg)

![12](https://user-images.githubusercontent.com/64316635/169139720-05eb2506-b8e6-4c9b-8fc5-bbcd161c30b3.jpg)

I just want to extract the clock and the current quarter:

1. Picture:  [1ST, 13:52]
2. Picture:  [1ST, 8:38]
3. Picture:  [1ST, 722]

I do not have a labeled test dataset so I thought of creating my own time and pictures of the quarter with 
[TextRecognitionDataGenerator](https://github.com/Belval/TextRecognitionDataGenerator) and replace the given time and quarter
with the small Picture of the timestamp and quarter.

![2475](https://user-images.githubusercontent.com/64316635/169141568-dfce0d1d-434e-4468-b501-f5be1e43e9f1.jpg)

I am already using the provided script for training.

But I have some questions about parameters for Training.

1. Would you use the described methode above to generate the testdataset or is there another simpler solution?
2. Does it matter that there is other text in the box which should not get recognized 
3. How many Training Pictures and Validation Pictures do I need for high accuracy results around 0.9
4. The training script provides a List of characters and symbols which should get detected? Should I just provide the necessary ones or do you think it is better to provide all characters
5. Which batch size and how many iterations would you suggest according to the amount of training data
6. Is there a possibility to let the modell know where in the picture to search or does the implemented CRAFT Modell already detect everything looking like a character or symbol

**Comment by rkcosmos** [comment]
If your aim is accuracy around 0.9, you can probably make it without training. Have you try vanilla model and a few argument setting ... something like, `allowlist = '0123456789STNDRTH:'`
